src/xdocs/property_types.xml

```diff
@@ -325,16 +325,55 @@
           <td>Definition</td>
         </tr>
 
+        <tr>
+          <td><code>alone</code></td>
```

Comment:
> alone and alone_or_singleline were moved on top without changes

Why? Issue states this is just for `alone_or_singleline`.
If we should order the values, we should order the types too.
Shouldn't this be a separate issue?

Reply:
I ordered values just to be logical to read:
alone
alone_or_singleline  is alone with nuances
same  is alone_or_singleline with nuances.

Reply:
Yes, but shouldn't this be a separate issue and re-order everything?

Reply:
We do not need reorder everything now.
We can make issue on this.
All I need is to clarify this particular property to make easy to read, sequential by meaning (it is just coincident that it looks like alphabetical order)